doc/source/starting-ray.rst

```diff
@@ -0,0 +1,105 @@
+Starting Ray
+============
+
+This page covers how to start Ray on your single machine or cluster of machines.
+
+.. contents:: :local:
+
+Installation
+------------
+
+Install Ray with ``pip install -U ray``. For the latest wheels (for a snapshot of ``master``), you can use these instructions at :ref:`install-nightlies`.
+
+Starting Ray on a single machine
+--------------------------------
+
+You can start Ray by calling ``ray.init()`` in your Python script. This needs to happen before any Ray tasks or actors are called (i.e., ``function.remote()`` will not work).
+
+.. code-block:: python
+
+  import ray
+  ray.init()
+
+To stop or restart Ray, use ``ray.shutdown()``. This is useful when Ray is initialized improperly:
```

Comment:
Not sure if we want the second sentence here. I don't really know what "initialized improperly" means and it's also somewhat scary for a user - makes me wonder if `ray.init()` just spuriously doesn't work

Reply:
```suggestion
To stop or restart Ray, use ``ray.shutdown()``.
```